### DISCUSSION TITLE
Migrating from binhex-radarr

### DISCUSSION THREAD

**Discussion by slydogsz** [RandomNinjaAtk/docker-radarr-extended#38] [General]
Hello,
I currently have binhex-radarr & binhex-sonarr running on my unraid setup and they are working well with my plex and sabnzbd setup.  I am hesitant to just yank them out and go back through the setup process again as it took me some time with my complex file system setup.

Is there a way that I can either implement your scripts in my current binhex- arr containers or migrate to your arr-extended containers without needing to go through setting everything back up?

I am disappointed that I didn't find this project until after I had already set up the binhex containers. I only discovered it after I started looking for a way to auto download trailers.

TIA

**Comment by RandomNinjaAtk** [comment]
You can pull the scripts from here and do what you want with them... But you'll have to account for required package dependencies and most likely need to run manually.
